tasks/system_probe.py

```diff
@@ -753,7 +753,7 @@ def replace_cgo_tag_absolute_path(file_path):
     for line in f:
         if line.startswith("// cgo -godefs"):
```

Comment:
Maybe you need to check for `cgo.exe -godefs` as well ?

Reply:
Yes. Good catch!